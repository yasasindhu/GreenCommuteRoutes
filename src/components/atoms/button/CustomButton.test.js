import React from 'react';
import ReactDOM from 'react-dom';
import { render, screen } from '@testing-library/react';
import userEvent from "@testing-library/user-event";
import Mybutton from './CustomButton'

describe("Button renders correctly", () => {
    test("should match snapshot", () => {
      const { asFragment } = render(<Mybutton />);
      expect(asFragment()).toMatchSnapshot();
    });
  
  });
  