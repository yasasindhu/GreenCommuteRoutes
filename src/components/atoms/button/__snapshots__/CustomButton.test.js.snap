// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly should match snapshot 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text"
    data-testid="button"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    />
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
